draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -659,35 +659,62 @@ the claims `given_name`, `family_name`, and `address`, as it would be sent from
 
 # Verification and Processing {#verification}
 
-## Processing by the Holder  {#holder_verification}
+## Verification of the SD-JWT {#sd_jwt_verification}
+
+Upon receiving an SD-JWT, a Holder or a Verifier MUST ensure that
+
+ * the Issuer-signed JWT is valid, i.e., it is signed by the Issuer and the signature is valid, and
+ * all Disclosures are correct, i.e., their digests are referenced in the Issuer-signed JWT.
 
-The Holder MUST perform the following (or equivalent) steps when receiving
-a Combined Format for Issuance:
+The Holder or the Verifier MUST perform the following (or equivalent) steps when receiving
+an SD-JWT:
+
+ 1. Separate the SD-JWT into the Issuer-signed JWT and the Disclosures (if any).
```

Comment:
```suggestion
 1. Separate the SD-JWT into the issuer-signed JWT, the Disclosures (if any), and the Key Binding JWT (if present). 
```
I do think the separation step needs to mention all of the parts.

Reply:
What a Holder does if it received a KB-JWT could also be covered here. Or elsewhere? Prior comment on the topic got marked as outdated for some reason https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/294#discussion_r1234263941 but I think I'd lean towards a should reject 

```suggestion
 1. Separate the SD-JWT into the issuer-signed JWT, the Disclosures (if any), and the Key Binding JWT (if present). If a Key Binding JWT is received by a Holder, the SD-JWT SHOULD be rejected.
```